### PR TITLE
Fix the issues with post-download upgrade

### LINKED ISSUE #3184: Iframe content sometimes isn't loading
This might be limited to Chrome 52.

Example page https://ampbyexample.com/advanced/how_to_create_interactive_amp_pages/

It is not 100% reproducible. When it fails the iframe itself is never requested.

### PR DESCRIPTION
**Pull request by dvoytenko**
Closes #3184.

### CODE REVIEW

File: src/custom-element.js [hunk 1]
```diff
@@ -1269,3 +1275,11 @@ export function resetScheduledElementForTesting(win, elementName) {
   delete knownElements[elementName];
 }
 
+/**
+ * Returns a currently registered element class.
+ * @param {string} elementName Name of an extended custom element.
+ * @return {?function()}
```
Comment (jridgewell) [inline src/custom-element.js]: `@visibleForTesting`

File: src/custom-element.js [hunk 2]
```diff
@@ -352,8 +354,11 @@ export function createAmpElementProto(win, name, implementationClass) {
     /** @private {?Element|undefined} */
     this.overflowElement_ = undefined;
 
+    // `opt_implementationClass` is only used for tests.
+    const ctor = opt_implementationClass || knownElements[name];
```
Comment (jridgewell) [inline src/custom-element.js]: `Ctor`, please.

**Comment by cramforce** [comment]
LGTM

File: src/custom-element.js (hunk 1 above)
Comment (dvoytenko) [inline src/custom-element.js]: Done for all such methods in this file.

Comment (jridgewell) [inline src/custom-element.js]: Thanks!

**Pull request merged**